readme.md

```diff
@@ -1120,7 +1120,11 @@ https://www.photographyblog.com/reviews/canon_powershot_sx70_hs_review/sample_im
 
 #### EOS RP
 
-https://www.dpreview.com/sample-galleries/0019055356/canon-eos-rp-sample-gallery/
+- https://www.dpreview.com/sample-galleries/0019055356/canon-eos-rp-sample-gallery/
+
+- raw.pixls.us (full(?) coverage, 8 samples: {C-RAW, RAW} x {16:9, 4:3, 3:2, 1:1}; CC0 licensed)
+
+  https://raw.pixls.us/data/Canon/Canon%20EOS%20R/
```

Comment:
oh wait, wrong link

Reply:
@lclevy can you just change that to https://raw.pixls.us/data/Canon/Canon%20EOS%20RP/ please?
i don't think it's worth a new PR